core/commands/destroy_command.rb

```diff
@@ -144,7 +147,7 @@ def destroy_by_configuration
         return
       end
     end
-    remove_files(configuration, @env.keep_template)
+    remove_files(configuration, @env.keep_template, @env.keep_configuration_path)
```

Comment:
Please remove modifications from remove files. You simply must not call this method if keep_configuration is defined.